container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/backend_creator/backend_creator.go

```diff
@@ -30,6 +42,30 @@ type APIContainerModeArgs struct {
 	APIContainerIP net.IP
 }
 
+var (
+	NoAPIContainerModeArgs *APIContainerModeArgs = nil
+)
+
+func GetDockerKurtosisBackend(
```

Comment:
This generic function creates a local or a remote docker backend based on the existence of a remote backend config.

Reply:
Same here, this is super interesting information, let's put it as a comment in the code directly